FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -633,22 +635,23 @@ def export_getComponentDocumentation( self, cType, system, module ):
     except Exception as _e:
       return S_ERROR( 'No documentation was found' )
 
-  types_storeHostInfo = []
-  def export_storeHostInfo( self ):
+  @staticmethod
+  def __storeHostInfo():
     """
     Retrieves and stores into a MySQL database information about the host
     """
-    result = self.__readHostInfo()
+    result = SystemAdministratorHandler.__readHostInfo()
     if not result[ 'OK' ]:
       gLogger.error( result[ 'Message' ] )
-      return S_ERROR( result[ 'Message' ] )
+      return result
 
     fields = result[ 'Value' ]
     fields[ 'Timestamp' ] = datetime.utcnow()
+    fields[ 'Extension' ] = fields[ 'Extensions' ]
```

Comment:
Yes. It is needed, because the db column called Extension. So, the Extension is never filled. I kept the backward compatibility as you proposed.